internal/services/compute/proximity_placement_group_resource.go

```diff
@@ -111,6 +153,22 @@ func resourceProximityPlacementGroupRead(d *pluginsdk.ResourceData, meta interfa
 		d.Set("resource_group_name", id.ResourceGroupName)
 
 		d.Set("location", location.Normalize(model.Location))
+
+		if props := model.Properties; props != nil {
+			if intent := props.Intent; intent != nil {
+				if intent.VMSizes != nil {
+					d.Set("intent_vm_sizes", set.FromStringSlice(*intent.VMSizes))
+				}
+			}
+		}
```

Comment:
this field always needs to be set into the state

Reply:
updated